### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Pessimistress**
For #1248

### CODE REVIEW

File: docs/get-started/get-started.md [hunk 1]
```diff
@@ -81,10 +81,51 @@ To use this API, consult Mapbox's [setRTLTextPlugin](https://docs.mapbox.com/map
...
 
 There's many other ready-to-run [examples](https://github.com/visgl/react-map-gl/tree/5.2-release/examples) you can take a look at if you need more inspiration.
+
+
+## Using with a mapbox-gl Fork
+
+Install react-map-gl along with your choice of fork along with react-map-gl, for example:
+
+```bash
+npm -i react-map-gl maplibre-gl
```
Comment (kylebarron) [inline docs/get-started/get-started.md:98]: `-i`? Should it be without the dash?
```suggestion
npm i react-map-gl maplibre-gl
```

**Review by kylebarron — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -29,34 +29,31 @@ npm install --save react-map-gl
 ### Example
 
 ```js
-import {Component} from 'react';
+import * as React from 'react';
```
Comment (ibgreen) [inline README.md:32]: The intro in line 17 above could be edited, I will put up a separate PR.

File: docs/get-started/mapbox-tokens.md [hunk 3]
```diff
@@ -1,20 +1,28 @@
 # About Mapbox Tokens
 
-react-map-gl and the underlying Mapbox GL JS libraries are open source and free to use. However, to load the map styles and tiles from Mapbox's data service, you will need to register on their website in order to retrieve an access token required by the map component, which will be used to identify you and start serving up map tiles. The service will be free until a certain level of traffic is exceeded.
+react-map-gl is open source and free. The underlying Mapbox GL JS library was free to use up to version 1.x, only requiring an access token if you load the map styles and tiles from Mapbox's data service. As of [v2.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v2.0.0), Mapbox GL JS must be used with an access token in order to access the map renderer.
```
Comment (ibgreen) [inline docs/get-started/mapbox-tokens.md:3]:
```suggestion
`react-map-gl` itself is open source and free. It is designed to provide React components for mapbox-gl-js or derived projects. 

Depending on which mabpox-gl-js fork you use, you may need a mapbox token. You will need a mapbox token if you use:
-  [Mapbox GL JS v2.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v2.0.0) requires a mapbox access token in order to access the map renderer, and generates billable events regardlesss of whether you are displaying your own maps.
-  [Mapbox GL JS v1.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v1.0.0) requires an access token only if you load the map styles and tiles from Mapbox's data service. If you display non-mapbox maps, you do not need a mapbox token.
```

**Review by ibgreen — APPROVED** [review]
Good start. I will put up a PR on the high level descriptions in README and docs/README.

File: docs/get-started/mapbox-tokens.md (hunk 3 above)
Comment (kylebarron) [reply docs/get-started/mapbox-tokens.md:3]: `mabpox`

Comment (Pessimistress) [reply docs/get-started/mapbox-tokens.md:3]: The module is called `mapbox-gl`. The official name is Mapbox GL JS.

**Review by alasarr — APPROVED** [review]

File: docs/get-started/mapbox-tokens.md (hunk 3 above)
Comment (kylebarron) [reply docs/get-started/mapbox-tokens.md]: I was trying to point out the `mabpox` typo in

> Depending on which mabpox-gl-js fork you use ...

File: docs/get-started/mapbox-tokens.md [hunk 4]
```diff
@@ -1,20 +1,32 @@
 # About Mapbox Tokens
 
-react-map-gl and the underlying Mapbox GL JS libraries are open source and free to use. However, to load the map styles and tiles from Mapbox's data service, you will need to register on their website in order to retrieve an access token required by the map component, which will be used to identify you and start serving up map tiles. The service will be free until a certain level of traffic is exceeded.
+`react-map-gl` itself is open source and free. It provides a React wrapper for `mapbox-gl` or derived projects.
+Depending on which `mabpox-gl` version (or fork) you use, you may need a Mapbox token. You will need a Mapbox token if you use:
```
Comment (kylebarron) [inline docs/get-started/mapbox-tokens.md:4]:
```suggestion
Depending on which `mapbox-gl` version (or fork) you use, you may need a Mapbox token. You will need a Mapbox token if you use:
```

**Pull request merged**